standalone/app/controllers/ItemController.scala

```diff
@@ -1,9 +1,14 @@
 package controllers
 
+import contentapi.ContentApiClient
+
 // If you add to this, don't forget the one in dev-build
-class ItemController(articleController: ArticleController, faciaDraftController: FaciaDraftController) extends ItemResponseController(
+class ItemController(articleController: ArticleController,
+                     faciaDraftController: FaciaDraftController,
+                     contentApiClient: ContentApiClient
+                    ) extends ItemResponseController(
   articleController,
-  GalleryController,
+  new GalleryController(contentApiClient),
```

Comment:
you could directly receive the controller as a parameter, as it is done for `articleController`, not that this has any sorts of impact as the controller shouldn't have any state